doc/source/ray-observability/reference/system-metrics.rst

```diff
@@ -19,7 +19,10 @@ Ray exports a number of system metrics, which provide introspection into the sta
      - Current number of tasks (both remote functions and actor calls) by state. The State label (e.g., RUNNING, FINISHED, FAILED) describes the state of the task. See `rpc::TaskState <https://github.com/ray-project/ray/blob/e85355b9b593742b4f5cb72cab92051980fa73d3/src/ray/protobuf/common.proto#L583>`_ for more information. The function/method name is available as the Name label. If the task was retried due to failure or reconstruction, the IsRetry label will be set to "1", otherwise "0".
    * - `ray_actors`
      - `Name`, `State`
-     - Current number of actors in a particular state. The State label is described by `rpc::ActorTableData <https://github.com/ray-project/ray/blob/e85355b9b593742b4f5cb72cab92051980fa73d3/src/ray/protobuf/gcs.proto#L85>`_ proto in gcs.proto. The actor class name is available in the Name label.
+     - Current number of actors in one of IDLE, or RUNNING states. The actor class name is available in the Name label.
+   * - `ray_actor_tasks`
+     - `Name`, `State`
+     - Current number of running tasks per actor in one of EXECUTING, RUNNING_IN_GET, or RUNNING_IN_WAIT states. The actor class name is available in the Name label.
```

Comment:
does this mean the class name is the name? or it's delimited somehow?

make docs more specific